### ISSUE TITLE
Notes without chart breaks the frontend

### ISSUE THREAD

**Issue by olafveerman** [NASA-IMPACT/covid-dashboard#291]
Notes without an accompanying chart break the secondary. We just added indicators to the API that have no chart, but do have notes. The result is a WSOD, with the following error message.

```
Uncaught TypeError: Cannot read property 'date' of undefined
    at sec-panel.js:128
    at Array.map (<anonymous>)
    at renderBody (sec-panel.js:124)
    at AccordionFoldCmp (accordion.js:153)
    at renderWithHooks (react-dom.development.js:14803)
    at mountIndeterminateComponent (react-dom.development.js:17482)
    at beginWork (react-dom.development.js:18596)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
```

We reverted the PR on the API that added the indicators to the groups. Once this is solved on the dashboard side, we should merge the PR again.

Example of an indicator with only a note: https://8ib71h0627.execute-api.us-east-1.amazonaws.com/v1/sites/sf

**Comment by olafveerman** [comment]
Related to #167

**Comment by vgeorge** [comment]
This should be fixed by #293, we just need to confirm it works with real data.

**Comment by olafveerman** [comment]
This is no longer the case with the latest frontend and API changes.
